terraform/github-my-repo.tf

```diff
@@ -3,3 +3,7 @@ module "github-repo-my-repo" {
   name        = "repo-name"
   description = ""
 }
+
+output "first-repo-full-name" {
+  value = "${module.github-repo-my-repo.full_name}"
```

Comment:
**[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Interpolation-only expressions are deprecated in Terraform v0.12.14